src/stories/Library/link-with-icon/LinkTypes.ts

```diff
@@ -0,0 +1,7 @@
+export type LinkTypes = "internal" | "external" | "download" | "search";
+
+export type IconTypes =
```

Comment:
Please change the type to be in singular form.

In general we do not use plural for types which contain a single value which is the case here.